### PR TITLE
Fixed MediaSelection Bugs

### PR DESCRIPTION
**Pull request by 3rfan**
…ction

| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

A bugfix for the `MediaSelection`.

#### Why?

Errors showed up when you pressed escape without opening the Overlay

### CODE REVIEW

**Comment by 3rfan** [comment]
@danrot

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
 // @flow
 import React from 'react';
 import classNames from 'classnames';
+import Loader from '../Loader';
 import type {Button as ButtonConfig} from './types';
 import Button from './Button';
 import headerStyles from './header.scss';
 
+const LOADER_SIZE = 24;
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js]: I guess you have clarified the size with the designer?

File: src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php [hunk 2]
```diff
@@ -114,6 +114,7 @@ function(Localization $localization) {
             (new Route('sulu_snippet.form.taxonomies', '/taxonomies', 'sulu_admin.list'))
                 ->addOption('resourceKey', 'snippets')
                 ->addOption('tabTitle', 'sulu_snippet.taxonomies')
+                ->addOption('adapters', ['table'])
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php]: This makes the PHP tests fail 🙈 Sorry, didn't think about that. You should add it to [this assertion](https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/SnippetBundle/Tests/Unit/Admin/SnippetAdminTest.php#L89).

**Comment by danrot** [comment]
And the label still only shows `Select Media` instead of the number.

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js (hunk 1 above)
Comment (3rfan) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Header.js]: No

**Comment by 3rfan** [comment]
@danrot update

**Comment by 3rfan** [comment]
@danrot now it's ready :)
Alt Text

**Pull request merged**